src/Tasks.java

```diff
@@ -0,0 +1,42 @@
+public class Tasks {
+
+    public static void taskNumberOne(int x, int y) {
+        int division = x / y;
+        int divisionRemainder = x % y;
+        System.out.println("Division result = " + division);
+        System.out.println("Remainder of the division = " + divisionRemainder);
+    }
+
+    public static void taskNumberTwo(int weight, int height) {
+        int perimeter = weight + weight + height + height;
+        int square = weight * height;
+        System.out.println("Perimeter = " + perimeter);
+        System.out.println("Square = " + square);
+    }
+
+    public static void taskNumberThree(int radius) {
+        double pi = 3.14;
```

Comment:
"Violation: 'NAME_CONSTANT' was not used for constant naming
Solution: Rename constant"